### PR TITLE
Fix various typos in documentation 

### PR DESCRIPTION
**Pull request by TheRandomMelon**
### Testing Done

### CODE REVIEW

File: docs/documentation/grid/tiles.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h2 class="subtitle">A <strong>single tile</strong> element to build 2-dimension
     <hr>
 
     <div class="content">
-      <p>To build intricate 2-dimensional, you only need a <strong>single element</strong>: the <code>tile</code>:</p>
+      <p>To build intricate 2-dimensional tiles, you only need a <strong>single element</strong>: the <code>tile</code>:</p>
```
Comment (jgthms) [inline docs/documentation/grid/tiles.html]: This should be "layouts".

File: docs/documentation/layout/footer.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
   <div class="container">
     <h1 class="title">Footer</h1>
     <h2 class="subtitle">
-      A simple responsive <strong>footer</strong> which can include anything: lists, headings, columns, icons, buttons...
+      A simple responsive <strong>footer</strong> which can include anything: lists, headings, columns, icons, buttons, etc...
```
Comment (jgthms) [inline docs/documentation/layout/footer.html]: It's either `etc…` or `etc.` but not both.

File: docs/documentation/overview/classes.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
 <section class="section">
   <div class="container">
     <h1 class="title">Classes</h1>
-    <h2 class="subtitle">Bulma is simply a <strong>collection</strong> of CSS classes. Write the HTML code you want.</h2>
+    <h2 class="subtitle">Bulma is simply a <strong>collection</strong> of CSS classes, so write whatever HTML code you want to write.</h2>
```
Comment (jgthms) [inline docs/documentation/overview/classes.html]: This change is not needed.

File: docs/index.html [hunk 4]
```diff
@@ -518,7 +518,7 @@ <h3 class="title is-2">
         <span class="icon">
           <i class="fa fa-book"></i>
         </span>
-        <span>Check the <strong>Docs</strong></span>
+        <span>Read the <strong>docs</strong></span>
```
Comment (jgthms) [inline docs/index.html]: This change is not needed.

File: docs/index.html [hunk 5]
```diff
@@ -421,7 +421,7 @@ <h3 class="title is-2">
           <i class="fa fa-arrows-v"></i>
         </span>
         <a href="{{ site.url }}/documentation/layout/hero/">
-          Easy <strong>vertical centering</strong> in <strong>fullscreen</strong>
+          Easy <strong>vertical centering</strong> in <strong>fullscreen</strong> mode
```
Comment (jgthms) [inline docs/index.html]: This suggests it's only available in native full screen mode.

File: docs/documentation/layout/hero.html [hunk 6]
```diff
@@ -52,7 +52,7 @@ <h2 class="subtitle">
   <div class="container">
     <h3 class="title">Colors</h3>
     <h4 class="subtitle">
-      As with buttons, you can choose one of the <strong>7 different colors</strong>
+        You can choose one of the <strong>7 available colors</strong>:
```
Comment (jgthms) [inline docs/documentation/layout/hero.html]: This change is not needed.

File: docs/documentation/grid/tiles.html [hunk 7]
```diff
@@ -341,7 +341,7 @@ <h3 class="title">Nesting requirements</h3>
       </div>
       <div class="message-body">
         <div class="content">
-          <p>You can however nest tiles more deeply than that, and mix it up!</p>
+          <p>You can nest tiles deeper than that, and mix it up!</p>
```
Comment (jgthms) [inline docs/documentation/grid/tiles.html]: Why lose the "however"?

File: docs/_layouts/documentation.html [hunk 8]
```diff
@@ -67,8 +67,7 @@
   <div class="container">
     <p class="has-text-grey-light">
       This page is <strong class="has-text-grey">open source</strong>.
-      Noticed a typo?
-      Or something unclear?
+      Noticed a typo? Or is something unclear?
```
Comment (jgthms) [inline docs/_layouts/documentation.html]: This change is not needed.

**Comment by TheRandomMelon** [comment]
Sorry for the lazy names of my commits, I just wanted to fix things up quickly lol

**Comment by TheRandomMelon** [comment]
OK, I've made all the changes you've requested @jgthms

**Pull request merged**